app/scripts/modules/core/pipeline/config/triggers/travis/travisTriggerOptions.component.ts

```diff
@@ -65,17 +65,22 @@ export class TravisTriggerOptions {
   };
 }
 
+class TravisTriggerOptionsComponent implements IComponentOptions {
+  public bindings: any;
+  public templateUrl: string;
+  public controller: any;
+
+  constructor() {
+    this.bindings = {
+      command: '='
+    };
+    this.templateUrl = require('./travisTriggerOptions.component.html');
+    this.controller = TravisTriggerOptions;
+  }
+}
```

Comment:
I don't know why, but I can't get the template to work if I don't require it and set it using `templateUrl`. Using your solution above, nothing is displayed and I get a `Error: Maximum call stack size exceeded.` in the browser console. Any tips?